module/settings.mjs

```diff
@@ -90,6 +90,16 @@ export default function registerSystemSettings() {
     type: Boolean
   });
 
+  // Use keyboard controls to set initiative advantage
+  game.settings.register("dnd5e", "initiativeAdvantageKeyboardControl", {
```

Comment:
I don't see why we need a setting to control this considering we don't have it for any other rolls (e.g. ability checks, attack rolls).

Reply:
Huh, I guess modules are patching in that behavior themselves. Maybe this would be better as a standalone module.